submit.go

```diff
@@ -570,6 +586,41 @@ func buildGithubIssueRequest(p parsedPayload, listingURL string) github.IssueReq
 	}
 }
 
+func (s *submitServer) sendEmail(p parsedPayload, reportDir string) error {
+	if s.cfg.SMTPServer == "" {
+		log.Println("Email submission disabled")
```

Comment:
feels like this is a bit redundant and will be a bit noisy on most servers?

Reply:
agreed, but I went for consistency because it does that for GH issues notifs or Slack one. Should I remove them on the way ?

Reply:
yeah, probably.